src/app/containers/FrontPageMain/__snapshots__/index.test.jsx.snap

```diff
@@ -982,6 +982,10 @@ exports[`FrontPageMain snapshots should render a pidgin frontpage correctly 1`]
       content="width=device-width, initial-scale=1, minimum-scale=1"
       name="viewport"
     />
+    <meta
```

Comment:
I'm doubting that my original suggestion (https://github.com/bbc/simorgh/issues/5158#issuecomment-575107725) is viable, as I'm not sure we should be adding this to every asset type. Worth checking with @lizcameron.

If not, I'd suggest reverting.